### ISSUE TITLE
Spring Server 6not working 

### ISSUE THREAD

**Issue by Myasir92** [swagger-api/swagger-codegen#10947]
I use the swagger json to create the node js server it generate successfully and also run but while using same json i try to generate spring server it shows error as shown below

Swagger Version:
2.4.28
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.springframework.cglib.core.ReflectUtils$1 (file:/C:/Users/HP/.m2/repository/org/springframework/spring-core/4.3.13.RELEASE/spring-core-4.3.13.RELEASE.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of org.springframework.cglib.core.ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

**Comment by HugoMario** [comment]
hey @Myasir92, i dont see errors but warning. Did you try running the generated code?

**Comment by Myasir92** [comment]
Yeah it run successfully but when i try to send request to any route so instead of specified host it send request to its own server not on the host and schema i define in json
